task/nexus-lifecycle-scan/0.1/nexus-lifecycle-scan-task.yml

```diff
@@ -0,0 +1,127 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: nexus-lifecycle-scan-task
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.33.2"
+    tekton.dev/categories: Cli, Continuous Integration, Developer Tools, Security
```

Comment:
I guess the no of allowed categories per resource is 2. I'll let @PuneetPunamiya answer this 😅

Reply:
@vinamra28 , I went ahead and reduced to two (2).